app/code/Magento/Integration/Model/Oauth/Consumer/Validator/KeyLength.php

```diff
@@ -39,7 +39,7 @@ class KeyLength extends \Zend_Validate_StringLength
      * Default encoding is set to utf-8 if none provided
      * New option name added to allow adding key name in validation error messages
      *
-     * @param  integer|array|\Zend_Config $options
+     * @param @inheritDoc
```

Comment:
Should be just `@inheritdoc`, no `@param` and all lower case

Reply:
@bbatsche That's what happens when you tip over the [ballmer peak](https://xkcd.com/323/) on a contribution day. Resolved.